### PR TITLE
FORUM-1382: Attachment icon in the forum topic doesn't disappear afte…

### PR DESCRIPTION
**Pull request by fmaatoug**
…r removal

### CODE REVIEW

File: forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java [hunk 1]
```diff
@@ -611,6 +611,7 @@ public void execute(Event<UITopicForm> event) throws Exception {
         }
       }
       uiTopicForm.refreshUploadFileList();
+      event.getRequestContext().addUIComponentToUpdateByAjax(uiTopicForm);
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java]: It works fine but we should refresh a smaller zone than the whole topic form. This refresh does an ugly flicking effect. We should reload only the attachments zone.

File: forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java [hunk 2]
```diff
@@ -611,6 +612,8 @@ public void execute(Event<UITopicForm> event) throws Exception {
         }
       }
       uiTopicForm.refreshUploadFileList();
+      UIComponent threadContent = uiTopicForm.findComponentById("ThreadContent");
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java]: Please use constant FIELD_THREADCONTEN_TAB instead.

File: forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java [hunk 3]
```diff
@@ -611,6 +612,8 @@ public void execute(Event<UITopicForm> event) throws Exception {
         }
       }
       uiTopicForm.refreshUploadFileList();
+      UIComponent threadContent = uiTopicForm.findComponentById("ThreadContent");
+      event.getRequestContext().addUIComponentToUpdateByAjax(threadContent);
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java]: Check for nullability on threadContent before using it.

Comment (fmaatoug) [reply forum/webapp/src/main/java/org/exoplatform/forum/webui/popup/UITopicForm.java]: Pr updated accordingly

**Pull request merged**